build/docker/Makefile

```
@@ -4,7 +4,7 @@ include $(ROOT)/deps/readies/mk/main
 
 REPO=redisearch
 
-REDIS_VERSION=7.2.0
+REDIS_VERSION=7.2.1
```

Maybe this is better so we won't have to change it upon every patch release? 
```suggestion
REDIS_VERSION=7.2-latest
```